engine/consensus/matching/sealingtracker/sealing_record.go

```diff
@@ -0,0 +1,97 @@
+package sealingtracker
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// SealingRecord is a record of the sealing status for a specific
+// incorporated Result. It holds information whether the result is sealable,
+// or what is missing to be sealable.
+// Not concurrency safe.
+type SealingRecord struct {
+	ExecutedBlock      *flow.Header             // block the incorporated Result is for
+	IncorporatedResult *flow.IncorporatedResult // the incorporated result
+
+	// SufficientApprovalsForSealing: True iff all chunks in the result have
```

Comment:
```suggestion
	// SufficientApprovalsForSealing: True if all chunks in the result have
```

Reply:
this is intentional and _not_ a typo 😉 :
> [[Wiki]](https://en.wikipedia.org/wiki/If_and_only_if): In logic and related fields such as mathematics and philosophy, **"if and only if" (shortened as "iff")** is a biconditional logical connective between statements, where either both statements are true or both are false.